templates/error_email.erb

```diff
@@ -1,9 +1,8 @@
 Hi <%= sqs_message['email'] %>
```

Comment:
I think we want this to be `<%= sqs_message['user_email'] %>` right?

Reply:
ah right! I missed this one. Thanks!